### PR TITLE
Fix and improve reactiveRangeCheckMode parameter documentation

### PR DESCRIPTION
**Pull request by jeandemanged**
**Please check if the PR fulfills these requirements**

- [x] The commit message follows our guidelines
- [x] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem?**

No

**What kind of change does this PR introduce?**

Doc fix & update

**What is the current behavior?**

`reactiveRangeCheckMode` is incorrectly saying that range at Pmax and Pmin is checked.

**What is the new behavior (if this is a feature change)?**
Accurate description of what `reactiveRangeCheckMode` actually does and better description of what is "reactive range".

**Does this PR introduce a breaking change or deprecate an API?**
- [x] No

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -86,7 +86,7 @@
 # List of patterns, relative to source directory, that match files and
 # directories to ignore when looking for source files.
 # This pattern also affects html_static_path and html_extra_path.
-exclude_patterns = ['_build', 'Thumbs.db', '.DS_Store']
+exclude_patterns = ['_build', 'Thumbs.db', '.DS_Store', 'README.md']
```
Comment (jeandemanged) [inline docs/conf.py:89]: this is just to remove this annoying warning at doc build

```
checking consistency... D:\powsybl-dev\powsybl-open-loadflow\docs\README.md: WARNING: document isn't included in any toctree
```

**Review by geofjamg — APPROVED** [review]

**Comment by annetill** [comment]
Reviewing the PR, I looked again in the code and I think we have a issue because the check on core side is not done https://github.com/powsybl/powsybl-core/pull/2563/, so for reactive capability curves, nothing ensures that max is greater than min...

File: docs/loadflow/parameters.md [hunk 2]
```diff
@@ -213,58 +213,67 @@ This parameter 'slows down' the Newton-Raphson by scaling the state vector betwe
...
+OpenLoadFlow discards voltage control for generators with a too small reactive power range, because in practice a too
+small reactive power ranger means limited to zero voltage control capability.
+
+For a given active power output, the reactive power range is defined as $MaxQ - MinQ$ (always a positive value).  
+The *maximum* and *minimum* reactive range of a generator is:
+- for generators without reactive limits: infinity 
+- for generators with reactive limits defined by a pair of [min/max values](inv:powsyblcore:*:*:#min-max-reactive-limits), both minimum and maximum reactive range are equal to $MaxQ - MinQ$ 
+- for generators with reactive limits defined by a [reactive capability curve](inv:powsyblcore:*:*:#reactive-capability-curve), the minimum (resp. maximum) reactive range is obtained by finding the curve point having the minimum (resp. maximum) $MaxQ - MinQ$.
+
+The `reactiveRangeCheckMode` parameter defines how generator reactive power range is to be tested:
```
Comment (annetill) [inline docs/loadflow/parameters.md:269]: Maybe you should add that if the test fails, the generator cannot control voltage. For me it is not clear.

Comment (jeandemanged) [reply docs/loadflow/parameters.md]: I agree, fixed

**Review by annetill — APPROVED** [review]

**Pull request merged**